code/modules/vehicles/motorized_wheelchair.dm

```diff
@@ -5,6 +5,7 @@
 	var/speed = 2
 	var/power_efficiency = 1
 	var/power_usage = 100
+	can_fold = FALSE
```

Comment:
Put redefined variables (without the `var/` part) before new ones, to keep the code a bit cleaner.